api-samples/contentSettings/contentSettings.md

```diff
@@ -0,0 +1,13 @@
+# chrome.contentSettings
+
+This sample demonstrates the use of `chrome.contentSettings` to display the settings of a given page in the extension's popup.
+
+## Overview
+
+The extension uses both the `chrome.contentSettings` and `chrome.contentSettings` APIs to check the value of each content setting on the user's currently active tab.
```

Comment:
Is this supposed to reference `get()` and `set()`?

A writing best practice is to use the strongest verb you can.

>The extension calls `chrome.contentSettings.get()` and `chrome.contentSettings.set()` to check...